Vagrantfile

```diff
@@ -105,7 +108,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     app.vm.hostname = "app"
     app.vm.network "private_network", ip: "192.168.8.24"
 
-    app.vm.synced_folder ".", "/opt/app", nfs: true
+    if testing?
+      app.vm.synced_folder ".", "/opt/app"
+    else
+      app.vm.synced_folder ".", "/opt/app", nfs: true
```

Comment:
In this case I don't actually know; we switched to NFS for HunchLab to solve a lot of stability issues with the default shared folders at the time, and we've tended to stick with it, but it's possible that the default shared folders have improved enough that NFS is no longer necessary. I may try removing nfs entirely if there are no objections, and see if it causes problems.